src/app/payment/services/payment.service.ts

```diff
@@ -189,17 +189,19 @@ const paymentService = {
     customerId: string,
     priceId: string,
     token: string,
-    mobileToken: string,
+    mobileToken?: string,
+    websiteToken?: string,
```

Comment:
This is wrong, in the use of the function in the `checkout.service.ts` you are passing only one token, here have added two, and passing the `currency` value as `websiteToken`